sdk/src/main/java/me/uport/sdk/extensions/TransactionExtensions.kt

```diff
@@ -79,7 +79,7 @@ fun Account.send(context: Context, to: String, value: BigInteger, callback: (err
 /**
  * Send contract call from account to [contractAddress] with [data] as the ABI encoded function call
  */
-fun Account.send(context: Context, contractAddress: String, data: ByteArray, callback: (err: Exception?, txHash: String) -> Unit) = GlobalScope.async {
+fun MetaIdentityAccount.send(context: Context, contractAddress: String, data: ByteArray, callback: (err: Exception?, txHash: String) -> Unit) = GlobalScope.async {
```

Comment:
I think these should remain `Account` extensions. They are not necessarily specific to `MetaIdentityAccount `

Reply:
@mirceanis Does this also apply to https://github.com/uport-project/uport-android-sdk/pull/89/files/0a070efee3b84bc8c3bcc52f6c2563c4f2082d4b#diff-fae0c63a2b1024643024fc9bfc833900